Sources/SwiftBSON/BSON.swift

```diff
@@ -128,137 +148,137 @@ public enum BSON {
 }
 
 /// Value getters
-extension BSON {
+public extension BSON {
```

Comment:
I just merged the formatting changes, so you can revert the changes to this file and rebase and the linting tasks should pass.